src/publish.ts

```diff
@@ -2,6 +2,7 @@ import fs from 'fs'
 import * as path from 'path'
 
 import globby from 'globby'
+import { template } from 'lodash'
```

Comment:
Lodash is a very heavy lib and overkill for this one case. 

Please solve this without using it.

Reply:
The goal is to have the same behavior than official semantic release plugins, and they already use `lodash`, people who use semantic release have already lodash as peer dependency, so I'm not sure to understand the point.

But indeed if you really want I can do with `.replace`, but it will be less modular and permissive

(https://github.com/semantic-release/git/blob/master/lib/prepare.js#L65)

Reply:
I change to import only template method, it's looks good for you ?